packages/mattermost-redux/src/selectors/entities/common.ts

```diff
@@ -51,3 +52,12 @@ export function getCurrentUserId(state: GlobalState): string {
 export function getUsers(state: GlobalState): IDMappedObjects<UserProfile> {
     return state.entities.users.profiles;
 }
+
+// Calls
+
+export function getCalls(state: GlobalState): Record<string, UserProfile[]> {
+    const callsPlugin = 'plugins-' + suitePluginIds.calls;
+
+    //@ts-ignore
```

Comment:
I had to add `//@ts-ignore` as `GlobalState` currently doesnt have `plugins-com.mattermost.calls` typed. Please let me know can I avoid this